integreat_cms/cms/migrations/0105_languagetreenode_languagetreenode_unique_lft_tree_and_more.py

```diff
@@ -1,8 +1,42 @@
 # Generated by Django 4.2.13 on 2024-09-25 15:01
 
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
 import django.db.models.constraints
 from django.db import migrations, models
 
+from ..utils.repair_tree import repair_tree
+
+if TYPE_CHECKING:
+    from django.apps.registry import Apps
+    from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+# pylint: disable=unused-argument
+
+
+def forwards_func(
```

Comment:
This does not attempt to fixes broken languagetreenode objects. I guess we can leave fixing that up to the users of the cms, given that it should be really unlikely?

Reply:
I think we did not experience any kind of compromised integrity with language tree nodes yet. But yes, ideally `repair_tree()` and the migration should also handle them. However, I'm not sure whether the extra effort is worth it